foundations/the_front_end/project_html_css.md

```diff
@@ -46,6 +46,25 @@ If you have entered `git push origin main` and typed out your username and passw
 
 Okay, that's enough Git for the moment -- time to actually build stuff!
 
+### Warning About Looking at Other People's Finished Project Code Before You have Completed Your Own
+
+All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to and see what other routes people have chosen to take with their project code and designs. 
```

Comment:
This is bad.

Reply:
Is that a test too, or is the wording actually bad and I need to change it